### PR TITLE
AC-19 send parameters

### PR DESCRIPTION
**Pull request by JonasPfeifer05**

### CODE REVIEW

File: src/fetch_agent/fetch_agent/agent.py [hunk 1]
```diff
@@ -32,6 +37,12 @@
 async def startup_event(ctx: Context):
     ctx.logger.info(f"Agent: {agent.name} ({agent.address})")
 
+    ctx.storage.set("open_time_frames", [(0, 0), (0, 0)])
```
Comment (leToxic) [inline src/fetch_agent/fetch_agent/agent.py]: imo should be in one set as JSON in pattern of PropertyResponse

File: src/fetch_agent/fetch_agent/agent.py [hunk 2]
```diff
@@ -70,6 +81,21 @@ async def on_register_car(ctx: Context, sender: str, msg: CarRegisterRequest):
     await ctx.send(sender, CarRegisterResponse(success=True))
 
 
+@agent.on_message(PropertyQueryRequest)
+async def on_query_properties(ctx: Context, sender: str, _msg: PropertyQueryRequest):
+    ctx.logger.info(f"{sender} requested params")
+
+    response = PropertyQueryResponse(
```
Comment (leToxic) [inline src/fetch_agent/fetch_agent/agent.py]: here you are using the Response. Why dont just save the Object and read the object

**Review by leToxic — DISMISSED** [review]
PropertyQueryResponse should be set in storage and not single dangling params

File: justfile [hunk 3]
```diff
@@ -7,8 +7,10 @@ init:
     echo "source ./install/local_setup.bash" >> ~/.bashrc
 
 install:
-    python3 -m pip install uagents==0.17.1
-    python3 -m pip install git+https://github.com/Diplomarbeit-PGHFP-2024-2025/aca-protocols.git@ffbf199cde07a21d41ff08e14450fae9594082cd
+    rm -rd venv/lib/python3.12/site-packages/aca_protocols
+    rm -rd venv/lib/python3.12/site-packages/aca_protocols-0.1.0.dist-info
```
Comment (ToBinio) [inline justfile]: works as temp fix see [ac-32](https://autocharge.atlassian.net/jira/software/projects/AC/boards/2?selectedIssue=AC-32)

**Review by xFizzar — DISMISSED** [review]
geo point not being sent when registering station

File: justfile [hunk 4]
```diff
@@ -24,3 +26,9 @@ build:
 
 run-agent:
     ros2 run fetch_agent agent
+
+all:
```
Comment (ToBinio) [inline justfile]: wtf

**Review by ToBinio — CHANGES_REQUESTED** [review]

File: justfile (hunk 3 above)
Comment (JonasPfeifer05) [reply justfile]: so its ok to keep this for now?

File: justfile (hunk 4 above)
Comment (JonasPfeifer05) [reply justfile]:
```suggestion
build-run-agent:
    just build
    just run-agent
```
better?

Comment (ToBinio) [reply justfile]: a lot

File: justfile (hunk 3 above)
Comment (ToBinio) [reply justfile]: yes

**Review by ToBinio — APPROVED** [review]

**Pull request merged**